### ISSUE TITLE
SwiftUI iOS 14 TextEditor

### ISSUE THREAD

**Issue by ferologics** [siteline/SwiftUI-Introspect#62]
Hi, I'm curious if it's possible to access the kerning attribute of the new TextEditor view.

**Comment by ethanbonin** [comment]
@ferologics, I've only tested this on iOS only.

```swift
import UIKit

extension View {
    func introSpecTextView(customize: @escaping (UITextView) -> ()) -> some View {
        return inject(UIKitIntrospectionView(
            selector: { introspectionView in
                guard let viewHost = Introspect.findViewHost(from: introspectionView) else {
                    return nil
                }
                return Introspect.previousSibling(containing: UITextView.self, from: viewHost)
            },
            customize: customize
        ))
    }
}
```

Example: 
```swift
TextEditor(text: $notesText.onChange(onChangeOfText))
    .introSpecTextView(customize: { (textView) in
        textView.becomeFirstResponder()
    })
```

**Comment by emreoktem** [comment]
Mac Version

```
import SwiftUI
import AppKit
import Introspect

extension View {
    func introSpecTextView(customize: @escaping (NSTextView) -> ()) -> some View {
        return inject(AppKitIntrospectionView(
            selector: { introspectionView in
                guard let viewHost = Introspect.findViewHost(from: introspectionView) else {
                    return nil
                }
                return Introspect.previousSibling(containing: NSTextView.self, from: viewHost)
            },
            customize: customize
        ))
    }
}
```

**Comment by JannThomas** [comment]
This has been solved #52.
